platform/broadcom/sai.mk

```diff
@@ -1,8 +1,8 @@
 BRCM_SAI = libsaibcm_4.3.3.4_amd64.deb
```

Comment:
Why this is libsaibcm_4.3.3.4_amd64.deb and NOT libsaibcm_4.3.3.4-1_amd64.deb ??

Reply:
Thanks for catching. Fixed it.